zerver/views/__init__.py

```diff
@@ -572,6 +572,36 @@ def dev_direct_login(request, **kwargs):
     return HttpResponseRedirect("%s%s" % (settings.EXTERNAL_URI_SCHEME,
                                           request.get_host()))
 
+@csrf_exempt
+@require_post
+@has_request_variables
+def api_dev_android_direct_login(request, username=REQ):
```

Comment:
I think REQ now only supports being used with parens (e.g. `username=REQ()`) -- we eliminated the old bare REQ syntax recently.